src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -107,6 +107,24 @@ Useful for multimodal co-registration with MEG, (S)EEG, TMS, and so on.
 | AnatomicalLandmarkCoordinates | RECOMMENDED           | [object][] of [arrays][] | Key:value pairs of any number of additional anatomical landmarks and their coordinates in voxel units (where first voxel has index 0,0,0) relative to the associated anatomical MRI (for example, `{"AC": [127,119,149], "PC": [128,93,141], "IH": [131,114,206]}`, or `{"NAS": [127,213,139], "LPA": [52,113,96], "RPA": [202,113,91]}`). Each array MUST contain three numeric values corresponding to x, y, and z axis of the coordinate system in that exact order. |
 
+### Echo-Planar Imaging and *B<sub>0</sub>* mapping
+
+Echo-Planar Imaging (EPI) schemes typically used in the acquisition of
+diffusion and functional MRI may also be *intended for* estimating the
+*B<sub>0</sub>* field nonuniformity inside the scanner (in other words,
+*mapping the field*) without the acquisition of additional MRI schemes
+such as gradient-recalled echo (GRE) sequences that are stored under the
+`fmap/` folder of the BIDS structure.
+
+The modality labels `dwi` (under `dwi/`), `bold`, and `sbref` (under `func/`)
+and any modality under `fmap/` are allowed to encode the MR protocol intent for
+fieldmap estimation using the following metadata:
+
+| **Key name**                | **Requirement level** | **Data type**                         | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| --------------------------- | --------------------- | ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| B0FieldIdentifier           | OPTIONAL              | [string][]                            | The presence of this key states that this particular 3D or 4D image MAY be used for fieldmap estimation purposes. The `B0FieldIdentifier` MUST be a unique string within one participant's tree, shared only by the images meant to be used as inputs for the estimation of a particular instance of the *B<sub>0</sub> field* estimation. It is RECOMMENDED to derive this identifier from DICOM tags, for example, the *Protocol Name* `(0018,1030)`.             |
```

Comment:
Something I forgot in my review- can't one image be a component of _multiple_ field map sets? Should this allow a list of strings? The same goes for `B0FieldSource`.

Reply:
Currently `B0FieldSource` admits a list of identifiers.

For `B0FieldIdentifier` - I don't think that's a common practice. By cases:

- Phases/phasediff/fieldmap - What would that mean? These are separate acquisitions so, from the workflow perspective, the only thing I would interpret is that the author want me to average them. You can achieve the same result by setting a list of identifiers to average in `B0FieldSource`.
- EPI (blip up/down) - Here it could have some consequence. But is there a reason to do this? Maybe if you want to fish for the best-looking correction exploring all the combinations possible of EPIs + DWI/BOLD/SBREF runs? Would you really need that from the encoder of a BIDS dataset? I can't really think of a use-case, but happy to be corrected.